src/popup/styles/settings/_dark.scss

```diff
@@ -0,0 +1,25 @@
+// Dark theme
+
+// Invert gray-scale colors
```

Comment:
Did you invert what was there, or are those the colors from [the styleguide](https://design.firefox.com/photon/visuals/iconography.html)?

Reply:
I inverted what was there in Bootstrap, as we're using that for the default theme and browsers other than Firefox. 🤔 I only adjusted the background color to be a more warm shade of gray which aligns better with what both Firefox and Chrome use.